known-issues-vs2017.md

```diff
@@ -164,117 +135,74 @@ If the DataContext class has a member (property, method, variable) defined with
+Manually find and install the package using Manage NuGet packages UI
 
-### Bower packages fail to restore
-Bower Restore fails to restore packages 
- 
-* #### Issue: 
-Bower Restore does not function, and IntelliSense for version numbers in bower.json always returns "*".  Attempt to restore displays the following in the Output Window (Bower/npm pane): 
+### Tag Helpers do not work
 
-  ECMDERR Failed to execute "git ls-remote --tags --heads https://github.com/lodash/lodash.git", exit code of #128 
+* #### Issue: 
+Razor Tag Helpers do not get colorization or special IntelliSense at design time.  They work normally at runtime. 
 
 * #### Workaround: 
-Install [the 12/12 update for Visual Studio 2017 RC](https://blogs.msdn.microsoft.com/dotnet/2016/12/12/updating-visual-studio-2017-rc-net-core-tooling-improvements)
+None available
 
-### Long time before all files are visible in Solution Explorer
-After creating a .NET Core or ASP.NET Core project, you need to wait until package restore completes 
+### TypeScript files do not compile
 
 * #### Issue: 
-As soon as creating a .NET Core or ASP.NET Core project, you will see a message in the status bar that it is restoring some packages. While this package restore is in progress, you will not be able to view all the files in Solution Explorer or build the project. If you tried to build a project while package restore is in progress, you might also see a dialog with the message "The build must be stopped before the project can be closed".  
+TypeScript files are not automatically compiled on save in .NET Core projects 
 
 * #### Workaround: 
-Install [the 12/12 update for Visual Studio 2017 RC](https://blogs.msdn.microsoft.com/dotnet/2016/12/12/updating-visual-studio-2017-rc-net-core-tooling-improvements)
+Add a tsconfig.json file to the root of the project, containing at least {}.
 
-### Entity Framework commands to not work
-Entity Framework Commands do not work in Package Manager Console 
+### Sequence numbers added to wronge item templates
 
 * #### Issue: 
-Entity Framework powershell command integration with msbuild is not yet complete, and Microsoft.EntityFrameworkCore.Design is omitted from ASP.NET Core Web Application with Individual Authentication project template. 
+In Core projects, item templates such as "npm Configuration File", "Bower Configuration File", "Grunt/Gulp Configuration File", etc., which work only with specific names, are created with an extra "1" inserted before the end of the file name. 
 
 * #### Workaround: 
-Install [the 12/12 update for Visual Studio 2017 RC](https://blogs.msdn.microsoft.com/dotnet/2016/12/12/updating-visual-studio-2017-rc-net-core-tooling-improvements)
+Edit the supplied file name before or after creating the file. 
 
-### Error running app with Windows Authentication
-Error running Core Apps using Windows Authentication 
+### Core Console/Class library  publish dialog not showing the right RID's
```

Comment:
This one needs to go to .NET Core Known Issues section and de-duped with the other one that's similar to this